boards/px4/fmu-v5/src/board_config.h

```diff
@@ -226,24 +222,12 @@
 #define GPIO_nARMED          /* PI0 */  (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_CLEAR|GPIO_PORTI|GPIO_PIN0)
 
 /* For protected build, define the LOCKOUT_STATE macros as function calls */
-#ifdef CONFIG_BUILD_FLAT
+#if defined(CONFIG_BUILD_FLAT)
 #define BOARD_INDICATE_EXTERNAL_LOCKOUT_STATE(enabled)  px4_arch_configgpio((enabled) ? GPIO_nARMED : GPIO_nARMED_INIT)
-#define BOARD_GET_EXTERNAL_LOCKOUT_STATE() px4_arch_gpioread(GPIO_nARMED)
+#define BOARD_GET_EXTERNAL_LOCKOUT_STATE() 				px4_arch_gpioread(GPIO_nARMED)
 #else
-static inline void board_indicate_external_lockout_state(bool enable)
-{
-	platformioclockoutstate_t state = {enable};
-	boardctl(PLATFORMIOCINDICATELOCKOUT, (uintptr_t)&state);
-}
-
-static inline bool board_get_external_lockout_state(void)
-{
-	platformioclockoutstate_t state = {false};
-	boardctl(PLATFORMIOCGETLOCKOUT, (uintptr_t)&state);
-	return state.enabled;
-}
-#define BOARD_INDICATE_EXTERNAL_LOCKOUT_STATE(enabled) board_indicate_external_lockout_state(enabled)
-#define BOARD_GET_EXTERNAL_LOCKOUT_STATE() board_get_external_lockout_state()
+#define BOARD_INDICATE_EXTERNAL_LOCKOUT_STATE(enabled) 	boardctrl_indicate_external_lockout_state(enabled)
+#define BOARD_GET_EXTERNAL_LOCKOUT_STATE() 				boardctrl_get_external_lockout_state()
```

Comment:
also here no need to extend the line lenght with n tabs...?

Reply:
Rest of the file tries to indent to the same TAB boundary. I think there are mixed single whitespace + TAB characters here now.